Sixteen750/src/main/java/org/firstinspires/ftc/sixteen750/subsystem/LiftSubsystem.java

```diff
@@ -1,227 +1,278 @@
 package org.firstinspires.ftc.sixteen750.subsystem;
 
+import java.util.function.Supplier;
+
 import com.acmerobotics.dashboard.config.Config;
 import com.acmerobotics.roadrunner.control.PIDCoefficients;
 import com.acmerobotics.roadrunner.control.PIDFController;
 import com.qualcomm.robotcore.hardware.DcMotorEx;
 import com.qualcomm.robotcore.util.Range;
+
 import com.technototes.library.hardware.motor.EncodedMotor;
-import com.technototes.library.logger.Log;
 import com.technototes.library.logger.Loggable;
 import com.technototes.library.subsystem.Subsystem;
 
-import org.firstinspires.ftc.sixteen750.Robot;
-
-import java.util.function.Supplier;
-
 @Config
+@SuppressWarnings("unused")
 public class LiftSubsystem implements Subsystem, Supplier<Double>, Loggable {
-    public static double TICKS_INCH = 136;
-
-    public static double INTAKE_POSITION_LIFT = 0;
-    public static double GROUND_JUNCTION_LIFT = 1.75 * TICKS_INCH;
-    public static double LOW_JUNCTION_LIFT = 14.5 * TICKS_INCH;
-    public static double MEDIUM_JUNCTION_LIFT = 25 * TICKS_INCH;
-    public static double HIGH_JUNCTION_LIFT = 36 * TICKS_INCH;
-    public static double MAX_HEIGHT = 38 * TICKS_INCH;
-    public static double MIN_HEIGHT = 0;
+    // Assuming the 0 position for both lift motor might be different
+    // The LiftSubsystem should be able to any of the motor combination
+    // Make everything right related private so don't take up space in FTC-Dashboard
```

Comment:
I'm not sure why you reverted the dual motor stuff. Are y'all going to be adding a motor back to the mix?